### DISCUSSION TITLE
Allow "Apps" to be either Typescript or WASM

### DISCUSSION THREAD

**Discussion by rj-au** [datalust/seq-tickets#1688] [Ideas]
At the moment, only .NET App can be installed.
It would be nice if we could upload/save our own TS/Node/WASM applications

**Comment by nblumhardt** [comment]
Hi Richard! Thanks for the suggestion. This is already possible with Seq today, as long as `node` or e.g. `wasmtime` is already installed on your Seq server/in the Seq container.

Instructions are in [this documentation article](https://docs.datalust.co/docs/seq-apps-in-other-languages) and there are a couple of examples in Rust out there (the [_Seq.App.JsonArchive_ app](https://github.com/datalust/seq-app-jsonarchive) is the best example to check out, I think).

Let me know if you need any more info!

**Reply by rj-au** [reply]
Super! thanks!

**Comment by rj-au** [comment]
This has been resolved
